test/report/merge.t

```diff
@@ -0,0 +1,26 @@
+Merge two files
+
+  $ echo "(lang dune 2.7)" > dune-project
+  $ cat > dune <<'EOF'
+  > (executables
+  >  (names test_merge1 test_merge2)
+  >  (instrumentation (backend bisect_ppx)))
+  > EOF
+  $ dune exec ./test_merge1.exe --instrument-with bisect_ppx
+  $ bisect-ppx-report summary --per-file
+   16.67 %   1/6   merge.ml
+  100.00 %   2/2   test_merge1.ml
+   37.50 %   3/8   Project coverage
+  $ dune exec ./test_merge2.exe --instrument-with bisect_ppx
+  $ bisect-ppx-report summary --per-file
+   33.33 %   2/6    merge.ml
+  100.00 %   2/2    test_merge1.ml
```

Comment:
This

```
   33.33 %   2/6    merge.ml
  100.00 %   2/2    test_merge1.ml
  100.00 %   2/2    test_merge2.ml
```

Looks wrong. I would expect

```
   16.67 %   1/6   merge.ml
  100.00 %   2/2   test_merge2.ml
```

It's probably because the second `bisect-ppx-report summary` is picking up the `.coverage` file from the first `dune exec`, and already implicitly merging them into the report, as it does.

Reply:
Ah sorry, I just re-read @arvidj's comment and understood that this is intentional :) Please disregard my comment.